### PR TITLE
[metadata manager] feat getMetadata

### PR DESCRIPTION
**Pull request by gandi0330**
This commits adds getMetadata function and command for that

I'm sorry I didn't use signature hooks. I'll use it next time

### CODE REVIEW

**Comment by chshin59** [comment]
I think `package.json` and `package-lock.json` should be excluded.

**Comment by gandi0330** [comment]
> I think `package.json` and `package-lock.json` should be excluded.

Oh, yeah. Thanks for letting me know

**Comment by gandi0330** [comment]
I reverted the dependencies of package.json and package-lock.json to the same as those of the existing project

**Comment by khlee025** [comment]
I think It's okay.

**Review by khlee025 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -406,6 +411,8 @@
     "@vscode/webview-ui-toolkit": "^0.9.3",
     "configparser": "^0.3.9",
     "ini": "^2.0.0",
+    "npm-watch": "^0.11.0",
+    "vscode": "^1.1.37",
```
Comment (chshin59) [inline package.json:415]: `$ npm uninstall npm-watch vscode`

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
-		"viewsWelcome": [
+    "viewsWelcome": [
```
Comment (chshin59) [inline package.json:19]: 나중에 지워주세요~

**Pull request merged**